src/sentry/conf/server.py

```diff
@@ -1783,4 +1784,8 @@ def get_sentry_sdk_config():
 # process_event might backlog and affect events from other platforms.
 SYMBOLICATOR_POLL_TIMEOUT = 4
 
-SENTRY_REQUEST_METRIC_ALLOWED_PATHS = ("sentry.web.api", "sentry.web.frontend", "sentry.api.endpoints")
+SENTRY_REQUEST_METRIC_ALLOWED_PATHS = (
```

Comment:
Black just decided to format this, I guess whoever committed it wasn't running black